components/build/openshift-pipelines/subscription.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: openshift-pipelines
   namespace: openshift-operators
 spec:
-  channel: pipelines-1.7
```

Comment:
@Michkov @stuartwdouglas perhaps this change cannot be made until other components sign off, but it was made during the kcp phase, and in fact @stuartwdouglas had to update jvm-build-service and bump its tekton dependencies as the API change affected its pipelinerun interactions

I can confirm without this change, the analyse-dependencies phase on the initial build fails with 

```
[maven-run : analyse-dependencies] Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: PATCH at: https://172.30.0.1/apis/tekton.dev/v1beta1/namespaces/ggmtest/taskruns/sample-component-build-flt95-maven-run/status. Message: admission webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming new object: json: unknown field "type". Received status: Status(apiVersion=v1, code=400, details=null, kind=Status, message=admission webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming new object: json: unknown field "type", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=BadRequest, status=Failure, additionalProperties={}).
```

otherwise, jvm-build-service will need to be changed to go back to the older tekton API